packages/react-components/src/components/Modal/components/ModalBase.tsx

```diff
@@ -69,7 +70,7 @@ export const ModalBase: React.FC<React.PropsWithChildren<ModalBaseProps>> = ({
   };
 
   return (
-    <div
+    <FloatingOverlay
```

Comment:
Have you checked how this works in AA? Does it clip with navbar or no?

Reply:
Yep, added `z-index` on the overflow wrapper to prevent elements overflow :)